### ISSUE TITLE
How to select a year

### ISSUE THREAD

**Issue by ivangodfather** [hons82/THCalendarDatePicker#28]
Hi, i want to use this calendar also for birthdays, theres a way to select a specific year without having to pass thoroug  all the months?

**Comment by hons82** [comment]
I'm not sure what you need.
If you just want to jump to a date in future change the date before launching it
```
self.datePicker.date = self.curDate;
```
there is no button or swipe or something if the calendar is open to switch year directly.

**Comment by ivangodfather** [comment]
Yeah, thats what i'm talking about, if i need to use it for select a birthday, so choosing past years like 1980 takes so long

**Comment by hons82** [comment]
Currently I'm not planing such a feature.
It's more a graphical challenge then a coding one.

Need to think about this

**Comment by hons82** [comment]
In V1.2.0 you'll find the solution to select the year faster.

**Comment by Diwann** [comment]
Hello, 
First, thank you a lot for this component, we use it @BlaBlaCar as our main date picker, and it's awesome. But we customized it a little bit (colors + customs images), and it doesn't fit well with the new year selection method. 
Could it be optional ?

**Comment by hons82** [comment]
Hi @Diwann,
Optional... Shouldn't be a big problem... Would it be enough to just hide the year selection buttons? Or is there any other recent change that doesn't fit?

I wasn't aware that @BlaBlaCar is using it. Nice. Could you add it [here](https://www.cocoacontrols.com/controls/thcalendardatepicker), as Apps that use this control.

**Comment by Diwann** [comment]
no big problem, just hiding the year selection buttons would be really enough. 
we just changed the asset to be more consistent with the overall design. 

FYI, here is what we have right now: 
![ios simulator screen shot 5 aout 2015 10 40 38](https://cloud.githubusercontent.com/assets/784023/9081629/77245d0e-3b5e-11e5-8df3-30c4d2ed26d1.png)

and here is what we see with the year selection button:
![ios simulator screen shot 5 aout 2015 10 38 54](https://cloud.githubusercontent.com/assets/784023/9081635/84c920f2-3b5e-11e5-8e80-e84701a0621a.png)

Concerning cocoacontrols, I had already tried but I can't add it to the users page because our app is not available on the usa and because cocoacontrols require an itunes url like : https://itunes.apple.com/us/app/* 
so this : https://itunes.apple.com/gb/app/id341329033 does not work ...

**Comment by hons82** [comment]
I've added the switch in the latest commit, but not fully tested.
This week I'm a bit short in time, so I don't know if I'm able to provide a new pod

**Comment by Diwann** [comment]
don't worry, no hurry
gonna test it tomorrow.

**Comment by Diwann** [comment]
ok, I found a small typo bug when selecting a previous month.
And the swipe recongnizer was allowing yearly swipe even with _disableYearSwitch enabled. 
( I made a PR : https://github.com/hons82/THCalendarDatePicker/pull/34 )

but I also found a strange bug : the first swipe on the calendar is showing something strange, which was not the case before, but I wasn't able to understang why. Here is a video to explain it : 
-> https://www.youtube.com/edit?o=U&video_id=i6viXJDSC8U
